tutorials/README.md

```diff
@@ -24,10 +24,13 @@ Our 2018 EMNLP tutorial is available at https://github.com/allenai/writing-code-
 
 ## How-to
 
-* [Using span representations](how_to/span_representations.md)
-* [Using pre-trained ELMo representations](how_to/elmo.md)
+* [Create your own Configuration File](how_to/create_a_configuration.md)
 * [How to train a model with lazy data](how_to/laziness.md)
+* [How to train and use a Transformer ELMo](how_to/training_transformer_elmo.md)
+* [How to use a Debugger](how_to/using_a_debugger.md)
```

Comment:
last nitpicks, i think this link title would be better as `How to Debug Your AllenNLP Code`.

Reply:
Done. 😊